src/data/exercises.json

```diff
@@ -0,0 +1,455 @@
+{
+    "total_count": 10,
```

Comment:
The JSON is a stub for the real thing that would return only the one 1 embed tag that is relevant.  I could put extra logic in to sort the "results" for items that match only the tag, but that would be unnecessary when using the real API.

I guess I put something in to handle both cases